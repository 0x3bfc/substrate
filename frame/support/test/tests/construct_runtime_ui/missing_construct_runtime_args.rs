use frame_support::construct_runtime;

mod system {
}

construct_runtime! {
	pub enum Runtime where
		Block = Block,
		NodeBlock = Block,
		UncheckedExtrinsic = UncheckedExtrinsic
	{
		System: system,
	}
}

fn main() {}
