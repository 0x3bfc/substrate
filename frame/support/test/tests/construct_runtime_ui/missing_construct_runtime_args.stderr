error[E0433]: failed to resolve: could not find `construct_runtime_args` in `system`
  --> $DIR/missing_construct_runtime_args.rs:12:3
   |
12 |         System: system,
   |         ^^^^^^ could not find `construct_runtime_args` in `system`
